src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -14,16 +14,33 @@
   </PropertyGroup>
 
   <Target Name="NpmRunBuild" BeforeTargets="BeforeBuild">
-      <Exec Command="npm install --prefix ./ @dynamods/splash-screen@latest" />
+      <Exec Command="npm pack @dynamods/splash-screen@latest" />
   </Target>
+
+    <Target Name="ExtractTGZFile" BeforeTargets="BeforeBuild">
+        <ItemGroup>
+            <TGZFiles Include="./*.tgz" />
+        </ItemGroup>
+
+        <ItemGroup>
+            <Reversed Include="@(TGZFiles->Reverse())" />
```

Comment:
this means?

Reply:
Reverts the order of the files to get the latest version

Reply:
Would this still work if this project is depending on more than one npm package?